salt/metalk8s/addons/monitoring/prometheus/exposed.sls

```diff
@@ -0,0 +1,28 @@
+{%- set kubeconfig = "/etc/kubernetes/admin.conf" %}
+{%- set context = "kubernetes-admin@kubernetes" %}
+
+Expose Prometheus:
+  metalk8s_kubernetes.service_present:
+    - name: prometheus
+    - namespace: monitoring
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
+    - metadata:
+        labels:
+          run: prometheus
+          prometheus: k8s
+        name: prometheus
+    - spec:
+        ports:
+        - name: api
+          port: 9090
+          protocol: TCP
+          node_port: 30222
+          targetPort: api
+        selector:
+          app: prometheus
+          prometheus: k8s
+        type: NodePort
+  require:
+    - pkg: Install Python Kubernetes client
+    - salt: metalk8s.addons.monitoring.prometheus.deployed
```

Comment:
These requires are useless here as it's a yaml key in `Expose Prometheus` and btw you should not require something not in the current sls (or in an included one)